package/ports_daemon/ports_daemon/src/ports.c

```diff
@@ -64,6 +77,20 @@ typedef int (*opts_get_fn_t)(char *buf, size_t buf_size,
 
 #define RUNIT_SERVICE_DIR "/var/run/ports_daemon/sv"
 
+static int opts_get_fixed_sbp(char *buf, size_t buf_size,
+                              const opts_data_t *opts_data)
+{
+  (void) opts_data;
+
+  struct stat s;
+  const char* tty = stat(RELEASE_LOCKDOWN, &s) == 0 ? "ttyGS1" : "ttyGS2";
```

Comment:
it's a bummer that we have to do these sorts of gymnastics but good catch on the lockdown enumeration issue and making it work!